src/libraries/System.ComponentModel.TypeConverter/src/System/ComponentModel/PropertyDescriptor.cs

```diff
@@ -212,15 +212,19 @@ protected override void FillAttributes(IList attributeList)
             base.FillAttributes(attributeList);
         }
 
+        [UnconditionalSuppressMessage("ReflectionAnalysis", "IL2026:RequiresUnreferencedCode", Justification = "No Attributes are supplied.")]
```

Comment:
@vitek-karas @MichalStrehovsky - this is a scenario where it would be really helpful if the library could define its own warning number for `RequiresUnreferencedCode`.

Today, passing an `Attribute[] filter` is what requires unreferenced code. But in the future, if `GetChildProperties` gets marked unsafe for a new reason, this suppression will still apply, and we won't get a warning.

So if I could mark `public virtual PropertyDescriptorCollection GetChildProperties(object instance, Attribute[] filter)` as `[RequiresUnreferencedCode(AttributeCollection.FilterRequiresUnreferencedCodeMessage, "SYSLIB12345")]`, then I could suppress just `SYSLIB12345` here. And if another `RequiresUnreferencedCode` gets added in the call stack in the future, this would start warning for the new warning.

Thoughts?